src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+    public static ArrayList<Tagging> allTagChanges = new ArrayList<Tagging>();
```

Comment:
recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
please check and differentiate the 2 diagrams in the activity description for better understanding